php/seatreg_functions.php

```diff
@@ -593,6 +593,21 @@ function seatreg_generate_settings_form() {
 				<textarea class="form-control" id="registration-info-text" name="registration-info-text" placeholder="<?php esc_html_e('Enter info text here', 'seatreg'); ?>"><?php echo esc_html($options[0]->info); ?></textarea>
 			</div>
 
+			<div class="form-group">
+				<label for="custom-footer-text"><?php esc_html_e('Custom footer text', 'seatreg'); ?></label>
```

Comment:
Seems like we don't need `for="custom-footer-text"` attribute on the label element. At least when I click on the label it wont autofocus editor